pipeline/authz/remote_json.go

```diff
@@ -84,6 +84,7 @@ func (a *AuthorizerRemoteJSON) Authorize(_ *http.Request, session *authn.Authent
 		return errors.WithStack(err)
 	}
 	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("Authorization", r.Header.Get("Authorization"))
```

Comment:
What happens when there is no `Authorization` header in the original request? Looks like it will still be set, then to an empty string?

Reply:
mmm, that's not so nice i guess. changed to only forward if authz header is non-empty

Reply:
can you maybe also add a test case for that? thanks :wink:

Reply:
done!